docs/content/manual/_incoming/2274-vm-import.md

```diff
@@ -61,23 +110,45 @@ vcsim   clusterReady
 ```
 
 ### Verify OpenstackSource
-* Define a secret for your Openstack cluster
-
-```
-apiVersion: v1
-kind: Secret
-metadata: 
-  name: devstack-credentials
-  namespace: default
-stringData:
-  "username": "user"
-  "password": "password"
-  "project_name": "admin"
-  "domain_name": "default"
-  "ca_cert": "pem-encoded-ca-cert"
-```
-
-* Define an OpenstackSource Object
+* Create a vm instance 
+1. Access the Openstack dashboard
+1. Create some images 
+![image](https://github.com/harvester/harvester/assets/29251855/dce72e41-7047-40dd-b549-2f479e173eee)
```

Comment:
lgtm. Just curious is there a convention not to store static files in this repo? I'm not sure how reliable the image link is in the future.

Reply:
Thanks for the suggestion. 
To provide better stability of image link, I tried adding static image files to test repo and update the url according to the relative path.